### DISCUSSION TITLE
A new voice assistant similar to this - help wanted

### DISCUSSION THREAD

**Discussion by Akul2010** [OpenVoiceOS/ovos-buildroot#134] [Q&A] [answered]
Hi, I just found this voice assistant and it looked cool. Can anyone find some time to help integrate some features from this voice assistant into mine? Link here: https://github.com/Akul-AI/akulai

**Comment by NeonDaniel** [comment, marked as answer]
I'm not sure what you have in mind for integration, but there is [documentation](https://openvoiceos.github.io/community-docs/) for all of the different OVOS modules. You can also use `ovos-core` as a dependency and extend any modules as you see fit which is what I do with [NeonCore](https://github.com/NeonGeckoCom/NeonCore).
